playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart

```diff
@@ -142,7 +144,18 @@ class ExamplesLoader {
   }
 
   Future<void> _loadOne(ExampleLoader loader) async {
-    final example = await loader.future;
+    Example example;
+    try {
+      example = await loader.future;
```

Comment:
```suggestion
      final example = await loader.future;
```
and continue the rest of the successful scenario within `try`.

Reply:
But that if something happened in `_playgroundController!.setExample`? In this case exception will be incorrect